tests/test_interactive_importer.py

```diff
@@ -1,8 +1,45 @@
 import unittest
 
+from bs4 import BeautifulSoup
+
 from gourmand.importers import interactive_importer
 
 
+class TestGetImages(unittest.TestCase):
+
+    def test_get_images(self):
+        self.html = """
+<!DOCTYPE html>
+<html dist-version="240" lang="fr">
+<body class="">
+<section class="item-page">
+<a class="navbar-brand" href="https://www.webtenerifefr.com/">
+        <img itemprop="logo" \
+            src="https://www.webtenerifefr.com/-/media/project/webtenerife/common/logos_tenerife/tenerife_logo_degradado_fra.svg" \
+                width="640" height="360" title="Turismo de Tenerife" alt="" />
+    </a>
+    <a class="navbar-brand-mobile" href="https://www.webtenerifefr.com/">
+        <img class="img-mobile" itemprop="logo" src="/dist/images/logo-tenerife-responsive.png" width="640" height="360"\
+              title="Turismo de Tenerife" alt="Turismo de Tenerife" />
+    </a>
+        <div class="footer__logo">
+                                    <img class="image--responsive" width="640" height="360" \
+                                        data-image-src="https://www.webtenerifefr.com/-/media/project/webtenerife/common/logos_tenerife/tenerife_logo_blanco_fra.\
+png" \
+                                        alt="Turismo de Tenerife" />
+                            </div>
+    </section>
+</body>
+</html>
+"""
+        self.soup = BeautifulSoup(self.html, "html.parser")
+
+        images = interactive_importer._get_images(self.soup)
+        self.assertIn('https://www.webtenerifefr.com/-/media/project/webtenerife/common/logos_tenerife/tenerife_logo_degradado_fra.svg', images)
```

Comment:
We know the exact list of URLs, thus please use `self.assertListEqual([...], images)`.